extensions/amp-ad-network-oblivki-impl/0.1/oblivki-a4a-config.js

```diff
@@ -0,0 +1,40 @@
+/**
+ * Copyright 2017 The AMP HTML Authors. All Rights Reserved.
```

Comment:
This is a new file, can you correct the year to `2020`?

```suggestion
 * Copyright 2020 The AMP HTML Authors. All Rights Reserved.
```